public/app/features/dashboard/components/GenAI/GenAIButton.tsx

```diff
@@ -7,11 +7,18 @@ import { Button, Spinner, useStyles2, Link, Tooltip } from '@grafana/ui';
 import { Message, generateTextWithLLM, isLLMPluginEnabled } from './utils';
 
 export interface GenAIButtonProps {
+  // Button label text
   text?: string;
+  // Button label text when loading
   loadingText?: string;
+  // Button click handler
   onClick?: (e: React.MouseEvent<HTMLButtonElement>) => void;
+  // Messages to send to the LLM plugin
   messages: Message[];
-  onReply: (response: string, isDone: boolean) => void;
+  // Callback when the LLM plugin responds. It is sreaming, so it will be called multiple times.
+  onGenerate: (response: string, isDone: boolean) => void;
```

Comment:
I renamed the function `onReply` to be consistent with the rest of the components.